src/main/webapp/linkWizard/LinkWizard.js

```diff
@@ -381,12 +336,13 @@ export const LinkWizard = hh( class LinkWizard extends Component {
           fileHandler: this.fileHandler,
           files: this.state.files,
           currentStep: currentStep,
-          existingConsentGroups: this.state.existingConsentGroups,
           consentGroup: this.state.consentGroup,
           updateForm: this.updateGeneralForm,
           projectKeyLabel: this.props.projectKey,
           consentGroupIsLoading: this.state.consentGroupIsLoading,
-          sampleCollectionIsLoading: this.state.sampleCollectionIsLoading,
+          getConsentGroupSampleCollections: this.props.getConsentGroupSampleCollections,
+          getConsentGroups: this.props.getConsentGroups,
+          unConsentedSampleCollections: this.props.unConsentedSampleCollections,
```

Comment:
All three of these are global variables ... why do they need to be properties passed into the component? Same thing goes for the other urls that are passed in as props. These should be available to the component regardless of how it is instantiated. In principle, everything that's in `main_component.gsp` should be accessible to any component. If that's not the case, there should be some refactoring done to make that happen.

Reply:
Yes, you're totally right. Also I just told the guys that we should change the way we were doing it. All that we need is present in `component.xxxx`